### PR TITLE
Add a new example using a retrieve method

### PR DESCRIPTION
**Pull request by joseribes**
Also a link to it in the Sample.md and for all the current examples adds a little description and a link to bigmlcom repo

### CODE REVIEW

File: Gists/ScoreAfterRetrieveAnomaly.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        {
+            // New BigML client with username and API key
+            Console.Write("user: "); var User = Console.ReadLine();
+            Console.Write("key: "); var ApiKey = Console.ReadLine();
+            var client = new Client(User, ApiKey);
+
+            // retrieve a anomaly detector with a known ID
+            Anomaly anomaly;
+            string anomalyId = "anomaly/54daa82eaf447f5daa000XXY"; //Put your ID here
+            while ((anomaly = await client.Get<Anomaly>(anomalyId)).StatusMessage.StatusCode != Code.Finished) await Task.Delay(10);
```
Comment (osroca) [inline Gists/ScoreAfterRetrieveAnomaly.cs]: GET is a synchronous request, do we really need a long polling process here?

Comment (joseribes) [inline Gists/ScoreAfterRetrieveAnomaly.cs]: El 15/07/2015 a las 11:10, Oscar Rovira escribió:
>
> In Gists/ScoreAfterRetrieveAnomaly.cs 
> <https://github.com/bigmlcom/bigml-csharp/pull/24#discussion_r34659550>:
>
> > +
> > +namespace Demo
> > +{
> > +    class ScoreAfterRetrieveAnomaly
> > +    {
> > +        static async void Main()
> > +        {
> > +            // New BigML client with username and API key
> > +            Console.Write("user: "); var User = Console.ReadLine();
> > +            Console.Write("key: "); var ApiKey = Console.ReadLine();
> > +            var client = new Client(User, ApiKey);
> > +
> > +            // retrieve a anomaly detector with a known ID
> > +            Anomaly anomaly;
> > +            string anomalyId = "anomaly/54daa82eaf447f5daa000XXY"; //Put your ID here
> > +            while ((anomaly = await client.Get<Anomaly>(anomalyId)).StatusMessage.StatusCode != Code.Finished) await Task.Delay(10);
>
> GET is a synchronous request, do we really need a long polling process 
> here?
>

No. Just copied & pasted code from other API request. Otherhand status 
checking should be kept.

Comment (osroca) [inline Gists/ScoreAfterRetrieveAnomaly.cs]: I know that the user has to check if the resource status is the right one (FINISHED) before trying to create a prediction, centroid or anomalyscore but make the code more complicate with the `while(....)` process is not worth.

**Comment by osroca** [comment]
Are you going to change the implementation of [the GET resource](#discussion_r34663177) I complained about in this PR? (`while(...)`)

**Comment by joseribes** [comment]
Changed in the last commit. Tested.

**Comment by osroca** [comment]
Merged!

**Pull request merged**